assets/bingo/lang/de_de.json

```
@@ -1,17 +1,286 @@
+	"bingo.lobby.tutorial.game_info.world_gen.heading": "Anpassungen Welt-Erzeugung",
+	"bingo.lobby.tutorial.game_info.world_gen": "Minecraft Bingo benutzt einige Anpassungen bei der Erzeugung.",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes.heading": "Winzige Biome: ",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes": "Biome sind kleiner, um die Erkundung zu erleichtern. Dadurch können auch andere Unterschiede im Biom-Layout auftreten.",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons.heading": "Mehr Verliese: ",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons": "Damit Verliese eine sinnvolle Option in einem schnellen Spiel sind, ist die Rate von Verliesen um den Faktor 12,5 erhöht. Das sollte es einfacher machen, sie bei jedem Durchlauf zu finden.",
+	"bingo.lobby.tutorial.game_info.world_gen.caves.heading": "Höhlen Anpassungen: ",
+	"bingo.lobby.tutorial.game_info.world_gen.caves": "Die Frequenz von Höhlen wurde leicht erhöht (von 1/7 in Vanilla auf 1/6).\n\nAußerdem gibt es keine überfluteten Höhlen in Ozeanen.",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts.heading": "Keine Minen: ",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts": "Minen haben sich als übermächtig gezeigt, da sie leichten Zufgriff auf viele Truhen geben. Minen sind daher ausgeschaltet in Bingo.",
+	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Keine *ite: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ites": "Alle Steinvarianten wie DiorIT, GranIT and AndesIT sind in dieser Version komplett entfernt.\nDadurch soll das Zumüllen des Inventars reduziert werden.",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient.heading": "Umgebungsbeleuchtung: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient": "Du kannst im Dunkeln besser sehen, aber immer noch hell von dunkel unterscheiden.",
+
+	"bingo.lobby.tutorial.get_tools.sign.line1": "Nach dem Start",
+	"bingo.lobby.tutorial.get_tools.sign.line2": "brauchst Du ",
+	"bingo.lobby.tutorial.get_tools.sign.line3": "erstmal ",
+	"bingo.lobby.tutorial.get_tools.sign.line4": "Werkzeug.",
+
+	"bingo.lobby.team_selection.sign.line1": "",
+	"bingo.lobby.team_selection.sign.line2": "Team",
+	"bingo.lobby.team_selection.sign.line3": "wechseln",
+	"bingo.lobby.team_selection.sign.line4": "",
+
+	"bingo.lobby.end.sign.line1": "Die Außenwelt",
+	"bingo.lobby.end.sign.line2": "",
+	"bingo.lobby.end.sign.line3": "Gefahr!",
+	"bingo.lobby.end.sign.line4": "Bleibt drinnen!",
+
+	"bingo.categories.header": "Kategorien:",
+	"bingo.category.apple": "Apfel",
+	"bingo.category.bow": "Bogen",
+	"bingo.category.bricks": "Ziegel",
+	"bingo.category.cactus": "Kaktus",
+	"bingo.category.cactus_bone": "Kaktus und Knochen",
+	"bingo.category.chest_iron": "Eisen und Truhen",
+	"bingo.category.chest_loot": "Verlies Beute",
+	"bingo.category.compass": "Kompass",
+	"bingo.category.diamond": "Diamant",
+	"bingo.category.egg": "Ei",
+	"bingo.category.nighttime_mob_drops": "Nacht Mobs Beute",
```

```suggestion
	"bingo.category.nighttime_mob_drops": "Nächtliche Monsterbeute",
```
No killing pugs xD (I just can't get over hearing "Mops" when talking about mobs in German using the English word)
Since "Nachtmonsterbeute" sounds stupid as well, changed "Nacht" to be an adjective.